libc/include/llvm-libc-types/jmp_buf.h

```diff
@@ -32,6 +32,9 @@ typedef struct {
 #elif defined(__riscv_float_abi_single)
 #error "__jmp_buf not available for your target architecture."
 #endif
+#elif defined(__arm__)
+  // r4, r5, r6, r7, r8, r9, r10, r11, r12, lr
+  long opaque [10];
```

Comment:
Extra space before `[` here.

This is already a problem for the x86-64 case, but it's not kosher to use plain identifiers here, or in any standard C header.  They need to be `__`-prefixed in case a user has `#define opaque syntax-error` before `#include <setjmp.h>`.  We should look into a clang-tidy check to enforce that.

Reply:
fixed in https://github.com/llvm/llvm-project/pull/93220/commits/77384336fb4ef69a615b59fc0152ffdc74a4fb4e